pkg/lib/operation/project/remote/job/run/operation.go

```diff
@@ -102,7 +102,7 @@ func (q *JobQueue) startLogRemaining() {
 			case <-q.done:
 				break
 			case <-time.After(time.Second * 5):
-				q.logger.Infof("Waiting for %s", strings.Join(q.getRemainingJobs(), ", "))
+				q.logger.Infof(`Waiting for "%s"`, strings.Join(q.getRemainingJobs(), `", "`))
```

Comment:
A tu som nasiel este jednu drobnost, vsade to ma `"`, ale tu nie, vid:
https://github.com/keboola/developers-docs/pull/248